sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatAsyncClient.java

```diff
@@ -118,48 +122,38 @@ public ChatThreadAsyncClient getChatThreadClient(String chatThreadId) {
     }
 
     /**
-     * Gets the list of chat threads in the first page.
+     * Gets the list of chat threads of a user.
      *
-     * @return the {@link CompletableFuture} that emits the first page of chat threads.
+     * @return the paged stream of chat threads of a user.
      */
-    @ServiceMethod(returns = ReturnType.SINGLE)
-    public CompletableFuture<Page<ChatThreadItem>> getChatThreadsFirstPage() {
-        ListChatThreadsOptions listThreadsOptions = new ListChatThreadsOptions();
-        return this.getChatThreadsFirstPage(listThreadsOptions, null)
-            .thenApply(response ->  new PageImpl<>(response.getValue(), response.getContinuationToken()));
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedAsyncStream<ChatThreadItem> listChatThreads() {
```

Comment:
I notice there're 2 functions for listChatThreads
PagedAsyncStream<ChatThreadItem> listChatThreads()
PagedAsyncStream<ChatThreadItem> listChatThreads(ListChatThreadsOptions listThreadsOptions, Context context)

I'll add another 
PagedAsyncStream<ChatThreadItem> listChatThreads(ListChatThreadsOptions listThreadsOptions)

Does it make sense?

Reply:
Guidelines say we will have an options bag in the maximal overload of a method, including context, so we don't really need to add another overload only for the options :)